### PR TITLE
Update rpl.c and uip.c (in ipv4)

### PR DESCRIPTION
**Pull request by skaterdude**
Very minor grammar / spelling edits to comments

### CODE REVIEW

File: core/net/rpl/rpl.c [hunk 1]
```diff
@@ -139,7 +139,7 @@ rpl_purge_routes(void)
       PRINTF("No more routes to ");
       PRINT6ADDR(&prefix);
       dag = default_instance->current_dag;
-      /* Propagate this information with a No-Path DAO to preferred parent if we are not a RPL Root */
+      /* Propagate this information with a No-Path DAO to preferred parent if we are not an RPL Root */
```
Comment (adamdunkels) [inline core/net/rpl/rpl.c]: RPL is supposed to be pronounced "ripple", so it should be "a RPL root" and not "an RPL root":

Comment (skaterdude) [inline core/net/rpl/rpl.c]: With all due respect, how would a reader know that!?

On January 4, 2015 6:30:16 PM EST, Adam Dunkels <notifications@github.com> wrote:
>> @@ -139,7 +139,7 @@ rpl_purge_routes(void)
>>        PRINTF("No more routes to ");
>>        PRINT6ADDR(&prefix);
>>        dag = default_instance->current_dag;
>> -      /* Propagate this information with a No-Path DAO to preferred
>parent if we are not a RPL Root */
>> +      /* Propagate this information with a No-Path DAO to preferred
>parent if we are not an RPL Root */
>
>RPL is supposed to be pronounced "ripple", so it should be "a RPL root"
>and not "an RPL root":
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/contiki-os/contiki/pull/936/files#r22443975

-- 
Sent from my Android phone with K-9 Mail. Please excuse my brevity.

**Comment by nvt** [comment]
It would be good to have the two "Update RPL" commits squashed, and the commit comment changed to be slightly more descriptive. This can be done using "git rebase -i <branch>".

File: core/net/rpl/rpl.c (hunk 1 above)
Comment (adamdunkels) [inline core/net/rpl/rpl.c]: It is impossible to know (that's why I thought I'd point it out) -- not sure if it even is in the RPL RFC :stuck_out_tongue_winking_eye: 

Good idea about adding it to the Wiki!

**Comment by mcr** [comment]
BTW: rfc6550 never says that RPL is pronounced "ripple", but it really OUGHT TO. I've entered an errata.  Love to have you all on the roll@ietf.org list...

**Comment by skaterdude** [comment]
Says "fatal: Needed a single revision ... invalid upstream HEAD-3" 

On January 5, 2015 3:52:23 PM EST, "Sébastien Dawans" <notifications@github.com> wrote:
>Assuming you want to squash you last 3 commits into 1,
>
>    git rebase -i HEAD~3
>
>It will open an editor with your last 3 commits preceded with the
>`pick` keyword. Replace the `pick` of lines 2 and 3 with `squash` or
>simply `s` then close the editor.
>
>It will open another editor, in which you can delete all commit
>messages and type a new one, or keep the one you prefer.
>
>    git push origin master -f
>
>You need to force push because you're rewriting the history of a branch
>you've already pushed.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/contiki-os/contiki/pull/936#issuecomment-68779042

-- 
Sent from my Android phone with K-9 Mail. Please excuse my brevity.

**Comment by gebart** [comment]
@skaterdude 
it's a tilde, not a minus character.
You can also try with single quotes as well to avoid the shell from doing crazy stuff there

    git rebase -i 'HEAD~3'

**Comment by cmorty** [comment]
Before everyone is explaining this over and over again: Some of us actually spent some time to write documentation: https://github.com/contiki-os/contiki/wiki/Code-Contributions#how-to-contribute-code And improvements are welcome!

**Comment by nvt** [comment]
Because @skaterdude still seems to have technical problems with the rebasing, I'll make an exception and merge this PR in its current state. :+1:

**Pull request merged**